pkg/cache/context_test.go

```diff
@@ -50,15 +50,54 @@ import (
 )
 
 const (
-	Host1  = "HOST1"
-	appID1 = "app00001"
-	appID2 = "app00002"
-	appID3 = "app00003"
-
-	pod1UID      = "task00001"
-	taskUID1     = "task00001"
-	pod1Name     = "my-pod-1"
+	Host1 = "host0001"
+	Host2 = "host0002"
+
+	appID1       = "app00001"
+	appID2       = "app00002"
+	appID3       = "app00003"
+	appID4       = "app00004"
+	appID5       = "app00005"
+	appIDUnknown = "app-none-exist"
```

Comment:
Use `non_existing_appID` to be consistent with `taskUnknown`

Reply:
Great. Update to `non_existing_appID` in the new commit.